### ISSUE TITLE
Color contrast is insufficient

### ISSUE THREAD

**Issue by AmyJuneH** [Drupal-NYC/drupalcampnyc#167]
Color contrast is insufficient on the User page. 
Example URL: https://2020.drupalcamp.nyc/user/9

The job title does not meet accessibility guidelines.

 WCAG -  1.4.3 Contrast (Minimum)
"The color of the text and the color of the background are not in sufficient contrast to each other.
The contrast ratio should be at least 4.5:1 for normal text and 3:1 for large text.
Note: Contrasts are not relevant where content is hidden. But make sure that for hidden content, which becomes visible when receiving focus, the contrast ratio is sufficient.
Insufficient contrast between text and its background can give problems for users with visual impairments and color blindness, but it can also affect a lot of other users, leading to important text potentially being overlooked."

Text color #5dcbfc / rgb(93,203,252) on Background color #ffffff / rgb(255,255,255) has the Contrast ratio
1.84 out of a required 4.5.

**Comment by jdleonard** [comment]
@AmyJuneH Please verify on Dev if you would (same credentials): https://dev-drupalcampnyc.pantheonsite.io/

**Comment by AmyJuneH** [comment]
@jdleonard Color contrast issue has been resolved.
Thanks!
